### DISCUSSION TITLE
Modal which has a form with Tabs has 2 column

### DISCUSSION THREAD

**Discussion by tbotee** [filamentphp/filament#14560] [Help] [answered]
### Package

Form builder

### Package Version

3.2

### How can we help you?

When tabs are presen in a modal, the modal has 2 columns and the form is in the first column. This is in a `ManageRelatedRecords` class and there s now custom view added.

<img width="996" alt="Screenshot 2024-10-18 at 12 32 25" src="https://github.com/user-attachments/assets/fdd87378-accc-4a86-b282-0bbeee55d5f5">
<img width="605" alt="Screenshot 2024-10-18 at 12 37 58" src="https://github.com/user-attachments/assets/ef0cab4d-f85b-403e-b529-0e1cc6a2d018">

**Comment by leandrocfe** [comment, marked as answer]
I think you can use

```php
return $form
    ->columns(null)
    ->schema(...)
    ...
```

**Reply by tbotee** [reply]
Thank you Leandro
